e2e/guest-links.spec.ts

```diff
@@ -72,6 +72,10 @@ test("creates a guest link and uploads a file as a guest", async ({
     await expect(guestPage.locator("h1")).toContainText("Guest Link Inactive");
     await expect(guestPage.locator(".file-input")).toHaveCount(0);
   }
+  await page.getByRole("menuitem", { name: "Files" }).click();
+  await expect(
+    page.locator("[test-data-filename]").nth(0).locator("td").nth(5)
+  ).toHaveText("Never");
```

Comment:
Can we refactor this to look more like the similar locator we use here?

https://github.com/mtlynch/picoshare/blob/c547d66f9db8844db1583043a54eb376a30c25f6/e2e/guest-links.spec.ts#L30-L32